buildSrc/src/main/groovy/docker-import-backup-lxc-data.gradle

```diff
@@ -21,6 +21,52 @@ Closure<Void> validateDatabaseBackupFileExtensions = {
 	return true
 }
 
+Closure<File> getFirstMatchingSubFolder
+
+getFirstMatchingSubFolder = {
```

Comment:
Just curious, any reason to split declaration and assignment in this case? Or just left over from an earlier state?

Reply:
Splitting the declaration and assignment is needed for recursion.